sklearn/ensemble/_gb.py

```diff
@@ -2521,6 +2549,22 @@ class GradientBoostingRegressor(RegressorMixin, BaseGradientBoosting):
 
     T. Hastie, R. Tibshirani and J. Friedman.
     Elements of Statistical Learning Ed. 2, Springer, 2009.
+
+    Examples
+    --------
+    >>> from sklearn.ensemble import GradientBoostingRegressor
+    >>> GradientBoostingRegressor(n_estimators=100, learning_rate=0.1,
+    ...     max_depth=1, random_state=0, loss='ls')
+    GradientBoostingRegressor(alpha=0.9, ccp_alpha=0.0,
+                          criterion='friedman_mse',
+                          init=None, learning_rate=0.1, loss='ls', max_depth=1,
+                          max_features=None, max_leaf_nodes=None,
+                          min_impurity_decrease=0.0, min_impurity_split=None,
+                          min_samples_leaf=1, min_samples_split=2,
+                          min_weight_fraction_leaf=0.0, n_estimators=100,
+                          n_iter_no_change=None, presort='deprecated',
+                          random_state=0, subsample=1.0, tol=0.0001,
+                          validation_fraction=0.1, verbose=0, warm_start=False)
```

Comment:
We need a better example here, maybe adapting one from the user guide. There is no need to print the estimator,
```py
>>> est = GradientBoostingRegressor(...).fit(X, y)
```
would be easier to read.